pages/testimony.tsx

```diff
@@ -1,15 +1,32 @@
 import { useRouter } from "next/router"
-import { Button, Spinner } from "react-bootstrap"
+import { Col, Row, Spinner } from "react-bootstrap"
 import { Container } from "../components/bootstrap"
 import {
   useBill,
   usePublicProfile,
   usePublishedTestimonyListing
 } from "../components/db"
 import { formatBillId } from "../components/formatting"
-import { Wrap } from "../components/links"
 import { createPage } from "../components/page"
 import { ViewAttachment } from "../components/ViewAttachment"
+import styled from "styled-components"
+
+const PositionSentence = styled(Container)`
+  background-color: white;
+  border-radius: 0.75rem;
+  padding: 1rem;
+  text-align: center;
+  font-weight: bold;
+`
+const Testimony = styled(Container)`
```

Comment:
We can add `white-space: pre-wrap;` here so the whitespace is preserved. 

https://digital-testimony-dev--pr612-revise-standalone-te-1mj7byp3.web.app/testimony?billId=H3340&author=CDxcKBZ6BjgetS4C2yuXbB5wgFC2